litellm/router.py

```diff
@@ -3334,6 +3344,17 @@ async def deployment_callback_on_success(
                     ttl=RoutingArgs.ttl.value,
                 )
 
+                ## RPM
+                rpm_key = RouterCacheEnum.RPM.value.format(
+                    id=id, current_minute=current_minute, model=deployment_name
+                )
+                await self.cache.async_increment_cache(
```

Comment:
could you please make sure this new op does not impact latency / perf @krrishdholakia 

In my tests adding 1 increment operation increase median latency by ~300ms at 100 RPS